webapp/src/components/App/Singlestudy/PropertiesDialog/index.tsx

```diff
@@ -70,7 +71,46 @@ function PropertiesModal(props: Props) {
     return tpmTagsChanged;
   }, [initTags, tags]);
 
-  const onSubmit = async () => {
+  usePromiseWithSnackbarError(
```

Comment:
You can remove `const [groupList, setGroupList] = useState<Array<GroupDTO>>([]);`

```javascript
const { data: groupList = [] } = usePromiseWithSnackbarError(
    getGroups, { errorMessage: t("settings.error.groupsError") }
);
```